lib/hsa/mcwamp_hsa.cpp

```diff
@@ -299,6 +299,12 @@ class HSAQueue;
 class HSADevice;
 } // namespace Kalmar
 
+
+// Global trackers for all HSA agents:
+// Possible these should move as member static in the future.
+static std::vector<hsa_agent_t > g_hsaAgents; // Array of the CPU + all GPU agents. 
```

Comment:
It seems `g_hsaAgents` is never read in any codes. Would future codes need this vector? If not we should consider remove it.